tests/unit/dom/blur-test.js

```diff
@@ -0,0 +1,68 @@
+import { module, test } from 'qunit';
+import { focus, blur, setContext, unsetContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+
+module('DOM Helper: blur', function(hooks) {
+  let element;
+
+  hooks.beforeEach(async function(assert) {
+    // used to simulate how `setupRenderingTest` (and soon `setupApplicationTest`)
+    // set context.element to the rootElement
+    this.element = document.querySelector('#qunit-fixture');
+
+    // create the element and focus in preparation for blur testing
+    element = buildInstrumentedElement('input');
```

Comment:
having `this.element` and `element` be two different things seems a little confusing 🤔

Reply:
I'm largely just trying to leverage the normal system of "rooting the selectors" that would happen in an app.  In those circumstances `this.element` (aka `getContext().element`) is the root element of the app / component test / etc. 

I think a good compromise here might be to use an explicit context _other_ than `this` which has the root element (`#qunit-fixture`) and to not call the local variable `element` here.

I'll work towards that...